lms/djangoapps/edraak_university/admin.py

```diff
@@ -17,7 +17,6 @@ class UniversityIDAdmin(admin.ModelAdmin):
         'edraak_user',
         'course_key',
         'university_id',
-        'section_number',
```

Comment:
- [ ] Could you just [exclude](https://docs.djangoproject.com/en/1.8/ref/contrib/admin/#django.contrib.admin.ModelAdmin.exclude) the unnecessary fields?